### PR TITLE
[hotfix] layernorm

### PR DESCRIPTION
**Pull request by kurisusnowdeng**
Allow layernorm without bias

### CODE REVIEW

File: colossalai/nn/layer/vanilla/layers.py [hunk 1]
```diff
@@ -254,3 +254,37 @@ def reset_parameters(self, weight_initializer, bias_initializer):
 
     def forward(self, input_: Tensor) -> Tensor:
         return F.linear(input_, self.weight, self.bias)
+
+
+@LAYERS.register_module
+class VanillaLayerNorm(nn.Module):
```
Comment (ver217) [inline colossalai/nn/layer/vanilla/layers.py:260]: Is it different from `torch.nn.LayerNorm`?

Comment (kurisusnowdeng) [reply colossalai/nn/layer/vanilla/layers.py:260]: @ver217 A little. We allow it to ignore bias.

**Review by ver217 — APPROVED** [review]

**Comment by feifeibear** [comment]
Do we have a unit test for layernorm?

**Comment by feifeibear** [comment]
Do we need a unitest for non-bias cases? But, if you have checked the updating, that's fine.

**Comment by kurisusnowdeng** [comment]
> Do we need a unitest for non-bias cases? But, if you have checked the updating, that's fine.

 @feifeibear Sure. We can refine the coverage of tests, as well as for the linear layers.

**Pull request merged**